FBC/FunctionalEntities/BusinessRegistries.rdf

```diff
@@ -342,15 +336,15 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>entity legal form registry entry</rdfs:label>
-		<skos:definition>an entry in an entity legal form registry that conforms to ISO 20275 and the Global Legal Entity Identifier Foundation (GLEIF) Entity Legal Form (ELF) code structure</skos:definition>
+		<skos:definition>entry in an entity legal form registry that conforms to ISO 20275 and the Global Legal Entity Identifier Foundation (GLEIF) Entity Legal Form (ELF) code structure</skos:definition>
 		<fibo-fnd-utl-av:abbreviation>ELF registry entry</fibo-fnd-utl-av:abbreviation>
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/code-lists/iso-20275-entity-legal-forms-code-list</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-fct-breg;EntityStatus">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-lif;LifecycleStage"/>
 		<rdfs:label>entity status</rdfs:label>
-		<skos:definition>a lifecycle stage indicating the operational and/or legal status of an entity as specified in a given business registry for that entity</skos:definition>
+		<skos:definition>lifecycle stage indicating the operational and/or legal status of an entity as specified in a given business registry for that entity</skos:definition>
```

Comment:
Does not need the business registry reference

Reply:
ok